ui/admin/app/routes/scopes/scope/aliases/index.js

```diff
@@ -79,28 +78,69 @@ export default class ScopesScopeAliasesIndexRoute extends Route {
           direction: sortDirection,
         };
 
+        const searchOptions = {
+          text: search,
+          relatedSearches: [
+            {
+              resource: 'target',
+              text: search,
```

Comment:
I think this `text` field can be removed since its not used in the query util since its also defined on line 82.